### PR TITLE
Add serialization of Decimal type for orjson.dumps in exporter

### PR DESCRIPTION
**Pull request by jjmurre**

### CODE REVIEW

File: src/schematools/exports/jsonlines.py [hunk 1]
```diff
@@ -15,6 +16,21 @@
...
+    raise TypeError
+
+
+def dumps(obj: Any) -> str:
+    """Json serializer.
+
+    Unfortunately, orjson does not support Decimal serialization,
+    so we need this extra function.
+    """
+    return orjson.dumps(obj, default=default)
```
Comment (lbam) [inline src/schematools/exports/jsonlines.py:31]:
```suggestion
def _default(obj: Any) -> str:
    if isinstance(obj, Decimal):
        return str(obj)
    raise TypeError


def _dumps(obj: Any) -> str:
    """Json serializer.

    Unfortunately, orjson does not support Decimal serialization,
    so we need this extra function.
    """
    return orjson.dumps(obj, default=_default)
```

File: src/schematools/exports/jsonlines.py [hunk 2]
```diff
@@ -34,7 +50,7 @@ def _get_row_modifier(self, table: DatasetTableSchema):
     def write_rows(  # noqa: D102
         self, file_handle: IO[str], table: DatasetTableSchema, sa_table: Table, srid: str
     ):
-        writer = jsonlines.Writer(file_handle, dumps=orjson.dumps)
+        writer = jsonlines.Writer(file_handle, dumps=dumps)
```
Comment (lbam) [inline src/schematools/exports/jsonlines.py:53]:
```suggestion
        writer = jsonlines.Writer(file_handle, dumps=_dumps)
```

**Comment by lbam** [comment]
LGTM but could use a unit test.

**Comment by jjmurre** [comment]
> LGTM but could use a unit test.

Good point. ATM, exporters do not have unittests at all, I'll add some.

**Review by barrydaniels-nl — APPROVED** [review]
Looks good

**Review by lbam — APPROVED** [review]

**Pull request merged**